web/new_alt_text_manager.js

```diff
@@ -142,6 +140,15 @@ class NewAltTextManager {
     });
 
     this.#overlayManager.register(dialog);
+
+    this.#learnMore.addEventListener("click", () => {
+      this.#currentEditor._reportTelemetry({
+        action: "pdfjs.image.alt_text.info",
+        data: {
+          topic: "alt_text",
+        },
```

Comment:
Nit: This fits on one line.
```suggestion
        data: { topic: "alt_text" },
```